packages/experimental/Button/src/Button.styling.ts

```diff
@@ -1,65 +1,31 @@
 import { buttonName, ButtonTokens, ButtonSlotProps, ButtonProps } from './Button.types';
 import { Theme, UseStylingOptions, buildProps } from '@fluentui-react-native/framework';
 import { borderStyles, fontStyles } from '@fluentui-react-native/tokens';
+import { buttonStates, defaultButtonTokens } from './ButtonTokens';
 
 export const stylingSettings: UseStylingOptions<ButtonProps, ButtonSlotProps, ButtonTokens> = {
-  tokens: [
-    (t: Theme) => ({
-      backgroundColor: t.colors.buttonBackground,
-      color: t.colors.buttonText,
-      borderColor: t.colors.buttonBorder,
-      borderWidth: 1,
-      borderRadius: 2,
-      disabled: {
-        backgroundColor: t.colors.buttonBackgroundDisabled,
-        color: t.colors.buttonTextDisabled,
-        borderColor: t.colors.buttonBorderDisabled,
-      },
-      hovered: {
-        backgroundColor: t.colors.buttonBackgroundHovered,
-        color: t.colors.buttonTextHovered,
-        borderColor: t.colors.buttonBorderHovered as string,
-      },
-      pressed: {
-        backgroundColor: t.colors.buttonBackgroundPressed,
-        color: t.colors.buttonTextPressed,
-        borderColor: t.colors.buttonBorderPressed as string,
-      },
-      focused: {
-        borderColor: t.colors.buttonBorderFocused,
-        color: t.colors.buttonTextHovered,
-      },
-    }),
-    buttonName,
-  ],
-  states: ['hovered', 'focused', 'pressed', 'disabled'],
+  tokens: [defaultButtonTokens, buttonName],
+  states: buttonStates,
   slotProps: {
     root: buildProps(
       (tokens: ButtonTokens, theme: Theme) => ({
         style: {
           display: 'flex',
-          alignItems: 'flex-start',
+          alignItems: 'center',
           flexDirection: 'row',
           alignSelf: 'flex-start',
+          justifyContent: 'center',
```

Comment:
I do wonder if some of these minHeight/Width or padding values should be tokens as well. Certainly the gap between the icon and content likely should be configurable.

Reply:
I went ahead and added minHeight/Width since they were straightforward.  I'm punting on Padding for now because it's a little less obvious how we should expose it in the token interface.